### ISSUE TITLE
IMHO: remapping } and { by default goes a bit to far

### ISSUE THREAD

**Issue by arecarn** [arp242/jumpy.vim#9]
Just a data point for you to consider. I use the built in behavior { and } quite a lot as movements, so remapping it by default is surprising, leaving it up to the user to configure { and } themselves is fine though.

Also consider the built in matchit plugin (see `:h matchit`) that remaps % to achieve similar results as { and }.

**Comment by arp242** [comment]
Yeah, I agree. I found the different semantics of `}` in Go files vs. other files to be confusing as well. I just haven't thought of a better default mapping for it yet 😅

> Also consider the built in matchit plugin (see :h matchit) that remaps % to achieve similar results as { and }.

It's not quite the same, and I've found configuring matchit to be rather byzantine. I just looked at it, and can't really figure out how to make it jump to the next/previous `if` in Go files. I also tried it in a C file (since `b:match_words` is set by default) and can't get it to work there either 🤔
